test/MUXControlsTestApp/App.xaml.cs

```diff
@@ -207,6 +207,10 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
             AppendResourceToMergedDictionaries("AdditionalStyles.xaml");
 #endif
 
+#if FEATURE_INFOBAR_ENABLED 
```

Comment:
This shouldn't be needed anymore, this was a hack we did to get the c# prototype to build. Could you also delete the PrototypePager one below?

Reply:
That's really weird, I thought I did remove those in the merge.